build-pipeline/dotnet-docker-linux-arm32v7-images.json

```diff
@@ -176,10 +198,18 @@
       "value": "$(Build.DefinitionName)-$(Build.BuildId)"
     },
     "image-builder.imageName": {
-      "value": "microsoft/dotnet-buildtools-prereqs:image-builder-jessie-20170817154316"
+      "value": "microsoft/dotnet-buildtools-prereqs:image-builder-jessie-20171004130549"
+    },
+    "image-builder.commonArgs": {
+      "value": "build --manifest manifest.json --repo microsoft/dotnet-nightly $(PB.image-builder.customCommonArgs)",
+      "allowOverride": true
     },
     "image-builder.args": {
```

Comment:
This name is confusing with the addition of the sdkArgs.  I think `image-builder.armBuildArgs` and `image-builder-sdkBuildArgs` would be more appropriate for the two.

Reply:
Clarification - you meant the dot separator as in `image-builder.sdkBuildArgs`?

Reply:
Sorry about that - yes I meant dot.